community/modules/scheduler/schedmd-slurm-gcp-v6-controller/README.md

```diff
@@ -74,6 +74,49 @@ page.
 More information on GPU support in Slurm on GCP and other HPC Toolkit modules
 can be found at [docs/gpu-support.md](../../../../docs/gpu-support.md)
 
+## Placement Max Distance
+
+When using
+[enable_placement](../../../../community/modules/compute/schedmd-slurm-gcp-v6-nodeset/README.md#input_enable_placement)
+with Slurm, Google Compute Engine will attempt to place VMs as physically close
+together as possible. Capacity constraints at the time of VM creation may still
+force VMs to be spread across multiple racks. Google provides the `max-distance`
+flag which can used to control the maximum spreading allowed. Read more about
+`max-distance` in the
+[official docs](https://cloud.google.com/compute/docs/instances/use-compact-placement-policies
+).
+
+You can use the `enable_slurm_gcp_plugins.max_hops.max_hops` setting on the
+controller module to control the `max-distance` behavior. See the following
+example:
+
+```yaml
+  - id: controller
+    source: ./community/modules/scheduler/schedmd-slurm-gcp-v6-controller
+    use: [ network, partition ]
+    settings:
+      enable_slurm_gcp_plugins:
+        max_hops:
+          max_hops: 1
+```
+
+> [!NOTE]  
+> `schedmd-slurm-gcp-v6-nodeset.settings.enable_placement: true` must also be
```

Comment:
should we consider adding this in the example above ?

Reply:
I vote no because 1 it is the default value and 2 then you have to add nodeset and partition to the example which makes it long. I think there are plenty of examples that show placement enabled.